sound/soc/sof/ipc4-topology.h

```diff
@@ -242,4 +242,11 @@ struct sof_ipc4_mixer {
 	struct sof_ipc4_msg msg;
 };
 
+struct sof_ipc4_src {
```

Comment:
can we add the struct descriptions to be consistent with the rest of the structs in this file?

Reply:
sure, thanks